google/cloud/pubsub/internal/publisher_stub.cc

```diff
@@ -129,6 +129,11 @@ std::shared_ptr<PublisherStub> CreateDefaultPublisherStub(
   // Newer versions of gRPC include a macro (`GRPC_ARG_CHANNEL_ID`) but use
   // its value here to allow compiling against older versions.
   channel_arguments.SetInt("grpc.channel_id", channel_id);
+  // Pub/Sub RPCs cannot contain more than 10MiB (this is the limit for a
+  // Publish() request), but often *do* contain more than the gRPC default of
+  // 4MiB.
+  channel_arguments.SetMaxSendMessageSize(16 * 1024 * 1024);
+  channel_arguments.SetMaxReceiveMessageSize(16 * 1024 * 1024);
```

Comment:
Q. How come we talk about 10MiB and 4MiB, but then use 16MiB?

Reply:
That was not very clear, was it?  I changed the comments and refactored the code to a single location. PTAL.